<!DOCTYPE html><!-- saved from url=(0014)about:internet --><html><head><meta http-equiv=Content-Type content="text/html; charset=utf-8"><meta http-equiv=x-ua-compatible content="IE=9"><title>Basic Lab Skills: Solutions and DNA Gel Electrophoresis</title><style type=text/css>body, td {
   font-family: sans-serif;
   background-color: white;
   font-size: 12px;
   margin: 8px;
}

tt, code, pre {
   font-family: 'DejaVu Sans Mono', 'Droid Sans Mono', 'Lucida Console', Consolas, Monaco, monospace;
}

h1 { 
   font-size:2.2em; 
}

h2 { 
   font-size:1.8em; 
}

h3 { 
   font-size:1.4em; 
}

h4 { 
   font-size:1.0em; 
}

h5 { 
   font-size:0.9em; 
}

h6 { 
   font-size:0.8em; 
}

a:visited {
   color: rgb(50%, 0%, 50%);
}

pre {	
   margin-top: 0;
   max-width: 95%;
   border: 1px solid #ccc;
   white-space: pre-wrap;
}

pre code {
   display: block; padding: 0.5em;
}

code.r, code.cpp {
   background-color: #F8F8F8;
}

table, td, th {
  border: none;
}

blockquote {
   color:#666666;
   margin:0;
   padding-left: 1em;
   border-left: 0.5em #EEE solid;
}

hr {
   height: 0px;
   border-bottom: none;
   border-top-width: thin;
   border-top-style: dotted;
   border-top-color: #999999;
}

@media print {
   * { 
      background: transparent !important; 
      color: black !important; 
      filter:none !important; 
      -ms-filter: none !important; 
   }

   body { 
      font-size:12pt; 
      max-width:100%; 
   }
       
   a, a:visited { 
      text-decoration: underline; 
   }

   hr { 
      visibility: hidden;
      page-break-before: always;
   }

   pre, blockquote { 
      padding-right: 1em; 
      page-break-inside: avoid; 
   }

   tr, img { 
      page-break-inside: avoid; 
   }

   img { 
      max-width: 100% !important; 
   }

   @page :left { 
      margin: 15mm 20mm 15mm 10mm; 
   }
     
   @page :right { 
      margin: 15mm 10mm 15mm 20mm; 
   }

   p, h2, h3 { 
      orphans: 3; widows: 3; 
   }

   h2, h3 { 
      page-break-after: avoid; 
   }
}</style><body><h1>Basic Lab Skills: Solutions and DNA Gel Electrophoresis</h1><p><strong>Safety Tip of the Day: Stay away from stuff that binds DNA!</strong></p><h3>Summary</h3><p>You might want to know how big some DNA fragments are. You can&#39;t measure them directly with a ruler, so you have to find a size related property that you can measure. Turns out that you can use electricity to drag the fragments through a porous medium, and the smaller the fragment, the farther it moves. If the fragments are dyed, you can actually measure how far they move and figure out how big they must be. The process is called Gel Electrophoresis. You are going to do it today.</p><h3>Learning Objectives</h3><p>Students will be able to:</p><ol><li>Describe the theory underlying gel electrophoresis and explain how it is used to separate macromolecules by size.</li><li><p>Demonstrate the following practical laboratory skills:</p><ul><li>Calculate mass/volume ratios to make up solutions of given concentrations.</li><li>Use a laboratory scale, including the tare function.</li><li>Measure volume with a graduated cylinder.</li><li>Use a micropipettor, including choosing the appropriate micropipettor depending on the volume(s) involved, reading and dialing for a specific volume, and pipetting up and dispensing specific volumes.</li><li>Calculate and set up a serial dilution.</li><li>Pour and load an agarose gel used for DNA analysis.</li></ul></li><li><p>Use R to create and plot a standard curve and perform a regression analysis.</p></li><li><p>Use a standard curve to determine the size (number of base pairs) of an unknown DNA fragment.</p></li></ol><h3>Preparation for Lab</h3><p>BEFORE CLASS, please read the following: <strong><em>NEEDS ADJUSTING TO NEW MATERIALS</em></strong></p><ol><li><p>This handout, including Supplement-Solutions at the end</p></li><li><p>Handouts in your Lab Manual Supplement (LMS) on:</p><ul><li>Micropipettors (B-1)</li><li>Balances (C-1)</li><li>Centrifugation (D-1)</li><li>Review Regression Analysis in Statistics (K-1), JMP (S-1), and Excel (J-1)</li><li>View videos on gels:</li></ul></li></ol><ul><li>Part 1 “Making and Agarose Gel” <a href="http://www.youtube.com/watch?v=wXiiTW3pflM&amp;feature=related">http://www.youtube.com/watch?v=wXiiTW3pflM&amp;feature=related</a></li><li>Part 2: “Running an Agarose Gel” <a href="http://www.youtube.com/watch?v=U2-5ukpKg_Q">http://www.youtube.com/watch?v=U2-5ukpKg_Q</a> .</li></ul><h2>Introduction</h2><p>In today&#39;s lab you will determine the size of an “unknown” piece of DNA.</p><ol><li><p>“Size” (or length) refers to how many pairs of nucleotides make up the piece of DNA. “Molecular weight” or “size” of DNA is typically reported in base pairs (bp).</p><p>A nucleotide consists of a 5-carbon sugar (deoxribose), a phosphate group, and a nitrogenous base. DNA is composed of 4 different nucleotides: Guanine, Cytosine, Adenine, or Thymine.</p><p>DNA is double-stranded. The individual nucleotides in a strand are linked to each other via strong covalent bonds. The two strands are held together by weaker hydrogen bonds (see Fig. 1). Within the double-stranded helix of DNA G always pairs with C (via 3 hydrogen bonds) and A always pairs with T (via 2 hydrogen bonds). Although an individual H-bond is weak, there are hundreds to thousands of such bonds stabilizing the two strands in a short piece of DNA and hundreds of millions of H-bonds in a human chromosome. In general, DNA is remarkably stable.</p></li><li><p>Agarose gel electrophoresis is a method used to separate nucleic acids (DNA and RNA) by size.</p><p>Agarose is a polysaccharide derived from seaweed. It can be mixed with water or buffer to form a gel, which is a type of porous matrix. The higher the concentration of agarose, the smaller the pores.</p><p>DNA is highly negatively charged, due to the phosphate groups that line the “backbone” of each strand. When an electric charge is applied to the agarose gel, DNA moves through the gel matrix toward the positive (anode) end (“opposites attract”). Smaller DNA fragments move more quickly through the porous medium. Larger fragments get hung up in the matrix and move more slowly.</p><p>The percent agarose used in the gel depends on the range of sizes you want to separate.</p><ul><li>1% agarose gels are good for separating DNA in the range of 1000 bp (or 1 kilobases, 1 kb) to 10 kb</li><li>2% agarose is better for separating DNA in the range of 100-1000 bp.</li></ul></li><li><p>DNA fingerprinting: an application of determining the size of DNA fragments</p><p>There are sequences in the human genome (and virtually the genomes of all organisms) that are highly repetitive. These sequences are called variable nucleotide tandem repeats or VNTRs. The repeated sequences are present in tandem (i.e. directly next to each other).</p><p>The number of repeats can vary from one individual to the next. The basic VNTR sequence is typically 12-20 base pairs long, and may be repeated on a chromosome 3-20 times (or more). For example, the basic unit of the VNTR called “D1S80” (yes, they all have such charismatic names) is 16 bp long. If it is repeated 10 times on one chromosome, the total length would be 160 bp.</p><p>Because humans have 2 copies of each chromosome (with the exception of the sex chromosomes in males), any one person will have 2 copies or alleles of a specific VNTR. For example, I may have inherited a version of D1S80 with 12 repeats from my father but another one with 15 repeats from my mother. My “genotype” could then be reported as D1S80 12,15. Someone else with different numbers of repeats for D1S80 would have a different genotype. This type of genotyping is used extensively in forensics and paternity testing.</p><p>How can we determine how many repeats someone has of a particular VNTR? You guessed it: agarose gel electrophoresis. By comparing a DNA band containing the repeat (obtained through other methods such as PCR) with a set of DNA size standards on a gel, we can determine the size, in base pairs, of the VNTR and thus the number of repeats. Thus, the size of an unknown length of DNA can be determined by generating a standard curve of DNA fragments of known size.</p></li><li><p>Today you will pour and run a gel that will be used to determine the size of an “unknown” piece of DNA. Standard curves are also commonly generated to determine the concentration of nucleic acids and proteins using different kinds of assays, something you will be doing in lab.</p></li><li><p>To begin learning the basics of gel electrophoresis, watch two really excellent videos on pouring and running a gel produced by the The University of Leicester. The gels shown in these videos are larger preparative-size gels whereas you will be working today with mini-gels. Still, the principles are all the same. The videos can be found on YouTube</p><ul><li>Part 1 “Making and Agarose Gel” <a href="http://www.youtube.com/watch?v=wXiiTW3pflM&amp;feature=related">http://www.youtube.com/watch?v=wXiiTW3pflM&amp;feature=related</a></li><li>Part 2: “Running an Agarose Gel” <a href="http://www.youtube.com/watch?v=U2-5ukpKg_Q">http://www.youtube.com/watch?v=U2-5ukpKg_Q</a> .</li></ul></li></ol><p>FIGURE 1 HERE</p><p><strong>Figure 1.</strong> <em>Structure of DNA.</em> The individual nucleotides on either strand are linked together via phoshodiester bonds, a covalent bond joining the 3’ carbon of one nucleotide with the phosphate group of the next nucleotide. Note the phosphate groups are negatively charged. The two strands of the DNA double helix are stabilized by hydrogen bonds between the nitrogenous bases; guanine (G) forms H-bonds with cytosine &copy; and adenine (A) with thymine(T). Illustration taken from the textbook Genetics: A Conceptual Approach (3rd ed) by Benjamin Pierce (Publ by WH Freeman &amp; Co.).</p><p>FIGURE 2 HERE</p><p><strong>Figure 2.</strong> <em>Visualization of 100bp Benchtop DNA ladder (Promega catalog # G8291) on 2% agarose gel stained with ethidium bromide.</em> 6 µL loaded per lane. The DNA ladder consists of 10 DNA samples that measure from 100 base pairs (bp) to 1000 bp in 100 bp increments, plus an additional 1500 bp fragment. This ladder serves as your molecular weight standards (i.e. you will be able to identify each band and its size on the gel after the gel has been run) which you can use to then identify the approximate size of other DNA samples run on the same gel, such as your “unknown” sample. When referring to DNA, “size” is the same as length, i.e. measured by the number of nucleotides (“base pairs”) that comprise the piece of DNA. Note the 500 bp band is brighter; it is present at higher concentration than other bands so that you can more easily distinguish it on your own gels. Photo taken from Promega online catalog (08/09). See <a href="http://www.promega.com/catalog/catalogproducts.aspx?categoryname=productleaf_1603">http://www.promega.com/catalog/catalogproducts.aspx?categoryname=productleaf_1603</a> for more details.</p><h2>Laboratory Exercise</h2><h4>Equipment and Materials Needed</h4><ol><li>Gel visualization/documentation system</li><li>Safety goggles (for observing gel with UV light source)</li><li>Laboratory balance</li><li>Small or medium sized weigh boats</li><li>Weighing spatula</li><li>125 mL erhlenmeyer flask</li><li>50 or 100 mL graduated cylinder</li><li>Microwave</li><li>Micropipettors (P20, P200, P1000 or equivalents)</li><li>Pipette tips</li><li>Disposal container</li><li>MiniGel electrophoresis rig, also mold, 8 lane comb, and gel-pouring gasket</li><li>Power supply</li><li>Microtube rack</li><li>6 sterile 0.5 µL microtubes</li><li>Latex gloves</li><li>Fine-tip markers</li><li>Paper towels</li><li>6” ruler with accurate mm markings</li><li>Agarose</li><li>1x TAE (Tris-acetate-EDTA) running buffer (40mM Tris base, 20mM acetic acid, 1mM EDTA): also contains ethidium bromide!</li><li>Sterile water</li><li>Benchtop 100bp DNA Ladder (Promega G8291)</li><li>DNA sample of unknown size (but known concentration)</li><li>Blue/orange loading dye, 6x (Promega G1881)</li></ol><h4>Protocol</h4><ol><li><p>Put on gloves. Keep them on throughout the procedure.</p><p>Wearing gloves keeps chemicals off your hands, particularly the gel buffer which contains ethidium bromide, a DNA-binding dye. Anything that binds DNA can mutate it and function as a potential carcinogen. Today we are using an extremely small amount that poses little health hazard but, ERR ON THE SIDE OF CAUTION and learn good laboratory practice.</p><p>If your gloved hands do come in direct contact with a chemical or buffer, please take them off (turn inside out) and dispose of them immediately – don’t touch other surfaces with your contaminated gloves!</p></li><li><p>Determine how much agarose you need.</p><p>Percent concentration for the gel is the ratio of mass of agarose powder (in grams, or g) to volume of buffer (in milliliters, or mL).</p><p>Today you will make a 2% gel in 40mL of buffer. How much agarose do you need?</p><p>ANSWER: ____________g (Do not weigh until your answer is checked by a lab instructor or lab assistant.)</p></li><li><p>Measure the agarose.</p><p>The laboratory scale you will use today can measure accurately and precisely to a tenth of a gram. (For measuring smaller amounts you need an analytical scale. Both types of scale contain a TARE function. See “Balances” in the Supplemental Lab Manual).</p><p>If not already turned on, turn the scale ON.</p><p>Place a medium size weigh pan (~2 inches square) on the scale.</p><p>Press TARE to rezero the scale. (The scale should be placed in a part of the lab that does not have excessive airflow; if you have trouble getting the scale to re-zero or stabilize, perhaps it is directly under an air vent.)</p><p>Measure out the appropriate amount of agarose (calculated in Step 2 above). Today, use a spatula to transfer agarose from the stock container to the weigh pan. If you add too much, remove some but do NOT place back into stock container. (For many applications, the stock container should be kept sterile, or at least uncontaminated; even the use of spatulas is unwise as is putting excess chemical back into the now-no-longer-sterile stock container.)</p><p>Label a 125 mL erhlenmeyer flask with your group ID and dump all of the agarose into it. Set aside.</p></li><li><p>Measure the buffer.</p><p>Measure out 40mL of 1X TAE buffer (in 8L carboy by sink in GATC lab) in a graduated cylinder. Measure to the bottom of the meniscus. (Measuring to the bottom of the meniscus is not especially important for the success of today’s lab but again, we are emphasizing good laboratory technique.)</p><p>Add the buffer to the agarose in the erhlenmeyer flask and swirl gently.</p></li><li><p>Melt the agarose.</p><p>Place a paper towel in the bottom of the microwave (to ease cleanup of any agarose solution that spills).</p><p>Place the flask containing your agarose solution on the paper.</p><p>Close the microwave door and begin microwaving at high setting for 2-3 minutes.</p><p>WATCH the flask EVERY SECOND and STOP the microwave the MOMENT the solution begins to boil! (Otherwise it will quickly boil over and you will lose much of your solution and have to start over.)</p><p>Remove the flask from the microwave using heat gloves or a wad of paper towels.</p><p>Gently swirl the contents of the flask so that the crystals of agarose are suspended. Be careful as the solution can begin to boil while you are swirling!</p><p>Place the flask back in the microwave and heat up again, again watching the flask very carefully. Once it has reached a ROLLING BOIL, turn off the microwave IMMEDIATELY and remove using heat gloves or a wad of paper towels. Avoid spilling (and spillovers).</p></li><li><p>Cool the gel solution.</p><p>Gently swirl the flask in a room temperature water bath or put on cool counter; occasionally remove glove and touch the glass with your hand. If you can keep your hand against the glass for several seconds, it is ready to pour.</p><p>Alternative method: Place the flask in a 60&deg;C water bath.</p><p>Cooling too little and cooling too much are both bad.</p><ul><li>If you cool too little, the solution might damage the polyacrylamide mold when you pour it in. The solution must be cooled to approximately 60&deg;C to prevent this from happening.</li><li>If you cool for too long, the gel begins to solidify in the flask rather than the mold.</li></ul></li><li><p>Add ethidium bromide to the gel.</p><p>Once the gel has cooled, add 3.0 μl of 10 mg/ml ethidium bromide (EtBr) solution to the gel.</p><p>Gently swirl the gel solution until the EtBr is thoroughly mixed.</p></li><li><p>Set up the mold.</p><p>Place the mold (or 2 molds at a time) in the gel caster. The function of the gel caster is to allow you to pour the gel into the mold so that it doesn’t leak.</p><p>Tighten the gaskets of the gel caster until “finger tight” but do NOT overtighten. (Over tightening causes the gasket to become deeply indented. Eventually, the gasket will become permanently deformed and unusable.)</p><p>Place the comb in the slots that are at the end of the mold closest to the immovable end of the gasket.</p></li><li><p>Pour the gel.</p><p>Once the gel is cooled enough, pour at a reasonably fast but steady pace into the middle of the mold</p><ul><li>slow enough to avoid air bubbles</li><li>fast enough to stop the gel from beginning to solidify before you are finished pouring.</li></ul><p>Remove any air bubbles around the comb with a pipet tip.</p><p>Leave the gel alone to solidify for 15-20 minutes. The gel will change from a clear solution to an opaque semi-solid similar in consistency to jello, but a bit sturdier.</p></li><li><p>Get 2 samples from your instructor.</p><ul><li>1 microfuge tube containing 14 µL of a DNA ladder, labeled “100bp ladder” (see Figure 1),</li><li>1 microfuge tube containing 15 µL of a DNA sample of “unknown” size, labeled “A”, “B”, etc. The unknown sample contains copies of a single gene (obtained through a PCR reaction) that is known to your instructor. Each unknown sample represents a different gene and is a different size, but all are between 300 and 1000 base pairs.</li></ul></li><li><p>Set up a two-fold serial dilution of your unknown.</p><ul><li>Line up 6 microfuge tubes and label with your unknown ID (e.g. “A” or “B”) and a dilution factor that identifies how much the original sample will be diluted: 1x, 2x, 4x, 8x, 16x, and 32x. (So, for example, the first tube in your series might be labeled “A-1x”).</li><li>Add 10 µL of your unknown sample to the first tube.</li><li>Add 5 µL of water to the remaining 5 tubes.</li><li>Take 5 µL from your first tube (the one with sample, labeled 1x) and add to your second tube (the one with 5 µL water and labeled 2x).</li><li>Mix well by flicking the tube several times.</li><li>Now take 5 µL of this 2x diluted sample and transfer to the third tube (labeled 4x).</li><li>Mix well and transfer 5 µL from this third tube to the next tube in the series.</li><li>Continue this dilution series until you reach the final tube.</li><li>Once you’ve mixed the final tube, remove 5 µL and discard.<br></li><li>All your tubes should now contain 5 µL total volume.</li></ul><p>Add 2.0 µL loading dye to each of your unknown samples. The loading dye contains a reagent that helps your sample to sink into the well; it also contains dyes that indicate how far samples have migrated into the gel during the run. For example, the yellow dye runs at ~50bp and the blue dye at ~300bp. (Your DNA ladder already contains loading dye.)</p><p>Mix all the tubes and briefly microfuge so that all samples are at the bottom of the tubes.</p></li><li><p>Load the gel.</p><p>Loosen the gasket and transfer the gel in its mold to a gel box. You may want to hold the gel mold between your thumb and 3rd or 4th finger, using your index finger to lightly hold the gel and prevent it from slipping out of the mold during transfer.</p><p>Add buffer to the gel box until the gel is completely submerged but no higher than the “high” mark on the side of the gel box.</p><p>Gently remove the comb by rocking back and forth. You want to avoid tearing or deforming the “wells” formed by the “teeth” of the comb.</p><p>Dial the micropipettor to 6 µL and load 6 µL of the 100bp DNA ladder into the first well (the one on the far left). You may want to steady the hand holding the pipettor with your other hand as well as rest both elbows on the countertop while loading.</p><p>Push down slightly on the plunger to remove any air in the tip of the pipet tip. Try not to poke the pipet tip into the gel at the bottom of the well. Also, do not push the pipettor down to the second stop to “blow out” the final drop as this can create an air bubble that blows half your sample out of the well.</p><p>Load 6 µL of DNA ladder into the far right lane (lane 8).</p><p>Load unknown samples 1-6 into lanes 2-7. All members of your group should take turns loading the gel. Loading the gel is a skill that could be tested during the lab practical exam.</p></li><li><p>Run the gel.</p><p>Once all the samples are loaded, carefully put the top on the gel rig on and plug into a power source. (<strong>The power source should be OFF when plugging in!</strong>)</p><p>Adjust the power supply so that the gel runs at 120V and hit the RUN button.</p><p>Check the gel about 5 minutes after you’ve turned it on to make sure the sample is running toward the “red” or positive/anode end of the gel. Then, let the gel run for 30-45 minutes, watching how far the front dye (the yellow one) has run into the gel.</p><p>Once the front dye has reached approximately &frac34; of the way into the gel, you stop the gel, turn off the power supply, remove the plugs, and carefully lift the top of the gel rig off.</p><p>Remove the gel in its mold. Be sure to support the gel with your index finger so that it doesn’t slip out of the mold.</p><p>Place mold on top of paper towels or in a small Tupperware container so that ethidium bromide-containing solution doesn’t drip on countertops or floor and transfer to UV light box for next step.</p></li><li><p>Visualize and document the gel run.</p><p>The ethidium bromide that stains the DNA is excited by UV light and emits light in the orange-red range. (See Introduction to Fluorescence and Spectroscopy Lab for more details.) You want to expose your gel to UV light but not your skin, eyes, etc, because you can get a severe burn. You can, however, view your DNA gel through a plexiglass screen that blocks UV rays. Your lab instructor will show you how to photograph your gel using the Photodocumentation system in the GATC lab.</p></li><li><p>Create a standard curve.</p><p>Print out a photo of your gel.</p><p>Using a ruler, measure the distance in millimeters (mm) from the bottom of the well to the leading edge (closest to the bottom of the gel) of each of the ladder fragments.</p><p>Enter the distance migrated and the DNA size for each ladder fragment into a Google Spreadsheet.</p><p>Make a labeled scatterplot of your data. For your analysis, the DNA size is your explanatory variable and the distance migrated is your response variable.</p><p>Notice from the scatterplot that DNA size is NOT linearly related to distance migrated. It turns out that the log of the molecular weight (i.e, size or length) of the DNA fragment is approximately linearly related to distance migrated.</p><p>Make a labeled scatterplot of log of the number of base pairs versus distance migrated. Your data points should approximately lie on a line (except possibly for the outermost points, the 100bp and 1500bp points.)</p><p>Do a logarithmic regression. Display and record in your notebook the regression equation, the R2 value, and the p-value.</p></li><li><p>Calculate the size of your unknown.</p><p>Measure the distance your unknown DNA band migrated and use this number and your regression equation to calculate its size (in base pairs).</p></li><li><p>Determine “limit of detection” of ethidium bromide stained gels.</p><p>Your instructor will provide you with the concentration (in nanograms per microliter, ng/µL) of your original (undiluted) unknown. When you examine your gel, you will notice that the more diluted your sample, the more difficult it is to see on the gel. Is there a specific concentration below which you cannot detect a band on the gel? What is it?</p><p>Examine the DNA ladder (either Figure 1 or your actual gel). Approximately the same number of molecules of the 100 bp DNA fragment are loaded as the 400 bp DNA fragment. Why does the 100 bp fragment appear fainter (less intense) than the 400 bp fragment?</p></li></ol><h3>CLEAN-UP</h3><ol><li><strong><em>Instructions need to be written.</em></strong></li></ol><h2>Assignment</h2><p>Turn in next week in lab a TYPED assignment, done as a group (names of all group members at top) with the following:</p><ol><li><p>A graph of your standard curve. Your figure should include</p><ul><li>Descriptive title</li><li>Labels for the axes</li><li>Scales on the axes. You may show the y-axis in either a linear or logarithmic scale.</li><li>Figure legend. (See Section N-3 of Lab Manual Supplement for contents of a figure legend)</li><li><strong><em>ARE THE INSTRUCTORS LOOKING FOR ANYTHING ELSE ON THE FIGURE? Do they want the data points as well as the standard curve?</em></strong></li></ul></li><li><p>Regression analysis for the standard curve, including</p><ul><li>Regression equation and what it means.</li><li>R2 and what it means.</li><li>p value and what it means.</li></ul></li><li><p>The ID name or number of your unknown sample, and its size (in base pairs).</p></li><li><p>Answers to the following 2 questions:</p><ul><li>What appears to be the limit of detection (in mg/mL) for this type of gel electrophoresis? In other words, what is the smallest concentration of DNA that you can visualize using this technique?</li><li>Why do smaller fragments appear fainter than larger fragments of similar concentration in the DNA ladder? <strong><em>IS THIS A CORRECT STATEMENT OF THE QUESTION?</em></strong></li></ul></li><li><p>Answers to the following 2 questions:</p><ul><li>How much agarose would you need to make up 50 mL of a 1.5% gel?</li><li>How much of a 40x TAE stock solution would you need to make up 8 liters of 1x TAE running buffer?<br>40x means “40 times the strength it is used at”. 1x commonly refers to the “working solution strength&quot;, i.e., the concentration that the solution is actually used at.</li></ul></li></ol><h2>SUPPLEMENT-SOLUTIONS</h2><h3>I. Metric system</h3><p>To calculate efficiently, you need a strong grasp of metric conversions. For most of what happens in a cell and genetics lab, you should be conversant with the basic units for</p><ul><li>volume (liter; L)</li><li>length (meter, m)</li><li>mass (grams; g)</li><li>concentration (molarity; M).<br></li></ul><p>In addition, you should be able to readily convert between the prefixes: milli-(m), nano-(n), micro-(µ), and pico-(p).<br>Practice makes perfect. Try your skill on these questions:</p><ul><li>How many µL are in 0.35 mL?</li><li>How many mg are in a g?</li></ul><h3>II. Pipetting and balances</h3><p>Each of the micropipets has been finely crafted to transfer very specific volumes (in µL). Section B-1 of your lab supplement contains a detailed description of the proper care and handling of these fine instruments. Each of the pipetment you will use costs ~$300. Please handle with care!</p><p>After carefully reading the lab supplement to become familiar with the proper use of the pipetmen and balances, ask one of the T.A.s to “quiz” you on your understanding. Once you have convinced a T.A. that you have mastered the use of micropipettors they will initial here: _____________________________. <strong><em>IF POSTED ONLINE, NEED NEW SYSTEM TO INITIAL</em></strong></p><h3>III. Concentration</h3><p>The skill of making solutions with specified concentrations underlies the successful completion of any experiment in the lab. There are two common ways to measure concentration.</p><ol><li><p>Molar concentration</p><ul><li>1 mole contains 6.02 x 1023 molecules of a compound.</li><li>A one molar (1.0 M) solution has one mole of a compound (solute) dissolved in a FINAL volume of 1 L of solvent.</li></ul></li><li><p>Percentage concentration</p><p>There are two common ways of defining percent concentration:</p><ul><li>mass/volume (m/v) More precisely, (mass of solute)/(volume of solvent)</li><li>volume/volume (v/v) More precisely, (volume of solute)/(volume of solvent)</li></ul><p>How would you make a solution that is 5% (v/v) of ethanol? <strong><em>WHAT IS THE SOLVENT?</em></strong></p><p>How would you make a solution that is 1% (m/v) of glucose? <strong><em>WHAT IS THE SOLVENT?</em></strong></p></li></ol><h3>IV. Calculation for making dilutions of stock solutions</h3><p>Often in the lab, your must make a dilution of a concentrated stock solution of a reagent. Calculations require the</p><pre><code> Dilution Equation:   c1 x v1= c2 x v2
</code></pre><p>where</p><ul><li><p>c1 is the known concentration of stock solution</p></li><li><p>c2 is the desired concentration of the reagent</p></li><li><p>v2 is the final volume of the buffer being prepared</p></li><li><p>v1 is the volume of the stock solution to use</p></li></ul>