<!DOCTYPE html>
<!-- saved from url=(0014)about:internet -->
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
<meta http-equiv="x-ua-compatible" content="IE=9" >

<title>Basic Lab Skills: β-Galactosidase Induction in Escherichia coli</title>

<style type="text/css">
body, td {
   font-family: sans-serif;
   background-color: white;
   font-size: 12px;
   margin: 8px;
}

tt, code, pre {
   font-family: 'DejaVu Sans Mono', 'Droid Sans Mono', 'Lucida Console', Consolas, Monaco, monospace;
}

h1 { 
   font-size:2.2em; 
}

h2 { 
   font-size:1.8em; 
}

h3 { 
   font-size:1.4em; 
}

h4 { 
   font-size:1.0em; 
}

h5 { 
   font-size:0.9em; 
}

h6 { 
   font-size:0.8em; 
}

a:visited {
   color: rgb(50%, 0%, 50%);
}

pre {	
   margin-top: 0;
   max-width: 95%;
   border: 1px solid #ccc;
   white-space: pre-wrap;
}

pre code {
   display: block; padding: 0.5em;
}

code.r, code.cpp {
   background-color: #F8F8F8;
}

table, td, th {
  border: none;
}

blockquote {
   color:#666666;
   margin:0;
   padding-left: 1em;
   border-left: 0.5em #EEE solid;
}

hr {
   height: 0px;
   border-bottom: none;
   border-top-width: thin;
   border-top-style: dotted;
   border-top-color: #999999;
}

@media print {
   * { 
      background: transparent !important; 
      color: black !important; 
      filter:none !important; 
      -ms-filter: none !important; 
   }

   body { 
      font-size:12pt; 
      max-width:100%; 
   }
       
   a, a:visited { 
      text-decoration: underline; 
   }

   hr { 
      visibility: hidden;
      page-break-before: always;
   }

   pre, blockquote { 
      padding-right: 1em; 
      page-break-inside: avoid; 
   }

   tr, img { 
      page-break-inside: avoid; 
   }

   img { 
      max-width: 100% !important; 
   }

   @page :left { 
      margin: 15mm 20mm 15mm 10mm; 
   }
     
   @page :right { 
      margin: 15mm 10mm 15mm 20mm; 
   }

   p, h2, h3 { 
      orphans: 3; widows: 3; 
   }

   h2, h3 { 
      page-break-after: avoid; 
   }
}

</style>



<!-- MathJax scripts -->
<script type="text/javascript" src="https://c328740.ssl.cf1.rackcdn.com/mathjax/2.0-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
</script>



</head>

<body>
<h1>Basic Lab Skills: β-Galactosidase Induction in Escherichia coli</h1>

<p><strong>Safety Tip of the Day: Don&#39;t touch the  <em>E. coli</em> and don&#39;t breathe the chloroform!</strong></p>

<h3>Summary</h3>

<p>The <em>lac</em> operon  is a segment of DNA in the bacteria \( E. coli \) consisting of  4 adjacent genes that are controlled together.  In the presence of lactose, the operon is turned ON and enables the bacteria to let lactose into the cell and break it down into glucose and galactose, nutrients for the cell. You can tell that the operon is ON by the the reduction in lactose, the increase in glucose or galactose, or the presence of β-galactosidase, the enzyme that breaks down the lactose, but all these are difficult to observe and measure. </p>

<p>Today you will measure the activity of the operon in a more indirect way.  You will add ONPG to your solution of <em>E. coli</em>.  ONPG is another molecule broken down by β-galactosidase.  One of its products is ONP, whose concentration is easily measured with a spectrophotometer. If ONP is present, the <em>lac</em> operon is ON and enabling the production of  β-galactosidase.  Your  will create a standard curve relating the concentration of ONP to absorbance (output of the spectrophotometer).  Using the standard curve you will measure an unkown concentration of β-galactosidase. </p>

<h3>Learning Objectives</h3>

<p><strong><em>Must be edited by biology dept</em></strong></p>

<p>Students will be able to:</p>

<ol>
<li> Write an experimental design, including detailed protocol.</li>
<li> Use R to create a graph summarizing results of an experiment.</li>
<li> Use R to analyze results of an experiment.</li>
<li> Prepare an abstract for a completed experiment.</li>
<li> Make an oral presentation on an experiment.</li>
<li> Do specific lab skills related to this specific experiment.
1 . Have theoretical understanding of topics relevant to this specific experiment</li>
</ol>

<h3>Preparation for Lab</h3>

<p><strong><em>TO BE WRITTEN taking into account newly developed materials</em></strong></p>

<h2>Introduction</h2>

<p>We will study the induction of the <em>lac</em> operon by monitoring the activity of an enzyme called β−galactosidase in <em>Escherichia coli</em> (<em>E. coli</em>).  Through this series of labs, you will gain experience designing experiments, collecting data, analyzing your results, and presenting your findings.  In addition, these experiments will require you to apply an understanding of several central principles in genetics and cell biology including:</p>

<ul>
<li> the ability of cells to perceive and respond to their environment</li>
<li> the flow of information from DNA→ RNA→ protein</li>
<li> the production of enzymes that function in cellular metabolism</li>
<li> the structure of “typical” prokaryotic gene clusters called “operons”</li>
<li> the regulated or inducible control of gene expression</li>
</ul>

<p><em>E. coli</em> is an organism that can survive in a wide variety of environments.  One mechanism that facilitates the survival of this prokaryotic organism in such divergent conditions is it’s capacity to rapidly and specifically produce sets of proteins that allow  a broad spectrum of organic molecules to be used as energy sources.  When a particular compound, such as a specific sugar or amino acid, is present in the surrounding medium individual <em>E. coli</em> cells recognize that these metabolites are available and respond by producing a set of proteins that facilitate the uptake and initial breakdown of these compounds.  Figure 1 shows how a protein whose expression is coordinated facilitates the breakdown of the disaccharide lactose. Lactose, or milk sugar, is a disaccharide composed of glucose and galactose linked together by a β-galactosidic bond.  <em>Galactoside permease</em> is a protein that transports lactose across the plasma membrane and <em>β-galactosidase</em> is a protein that functions as an enzyme to hydrolyze polysaccharides whose monomers are linked by β-galactosidic bonds. </p>

<p>FIGURE 1 HERE</p>

<p><strong>Figure 1.</strong>  <em>The catabolism of lactose.</em> The breakdown of the disaccharide lactos involves enzymes (boxes) whos <strong><em>conclusion is unreadable in the lab manual</em></strong></p>

<p>The goal of this multi-week series of experiments will be to determine whether treating <em>E. coli</em> cells with various sugars or amino acids stimulates the production (e.g. transcription and translation) of the enzyme β-galactosidase. You may want to look up additional information concerning DNA transcription and gene structure in your textbook or in other resources.</p>

<ul>
<li>A starting point: <a href="http://users.rcn.com/jkimball.ma.ultranet/BiologyPages/T/Transcription.html">http://users.rcn.com/jkimball.ma.ultranet/BiologyPages/T/Transcription.html</a></li>
<li>Another general starting point for biological tutorials: <a href="http://science.nhmccd.edu/biol/bio1int.htm">http://science.nhmccd.edu/biol/bio1int.htm</a> <strong><em>THIS LINK SEEMS TO BE BROKEN OR WRONG</em></strong>.)</li>
</ul>

<p>The so-called “<strong>Central Dogma</strong>” (DNA encodes RNA which is decoded during the translation of proteins) represents the most common flow of genetic information within a living cell.  Only 40 years ago the idea that the expression of a gene could be switched on and off was revolutionary.  In prokaryotes, the tight regulation of gene transcription (e.g. use of DNA to direct the synthesis of RNA) allows a cell to utilize available energy resources with great economy.  This efficient use of assets is important because there may be large fluctuations in the quantity and quality of nutrients available based on a bacteria’s environment.  As such, there are advantages to bacteria being able to quickly produce the enzymes (a type of protein) required to adapt to a changing environment. </p>

<p>Bacteria genomes contain clusters of genes that are referred to as <strong>operons</strong>.  The presence of linear, sequentially arranged genes <strong><em>SYNTAX PROBLEM - WHAT IS MEANT?</em></strong> into operons allows the coordinated production of sets of RNA and the proteins they encode.  Thus, prokaryotic cells can synchronize the production of suites of enzymes that function in a metabolic pathway to metabolize a particular compound in response to environmental cues or the availability of a compound. The <em>lac</em> operon codes for proteins required to transport the disaccharide lactose into the cell and to break it down.<br/>
Figure 2 shows how expression of the <em>lac</em> operon is regulated.</p>

<p>FIGURE 2 HERE</p>

<p><strong>Figure 2.</strong>  <em>Reulation of the lac operon.</em> Transcription of the <em>lac</em> operon is regulated by binding of the <em>lac</em> repressor &reg; to the operator sequence. </p>

<p>In an operon, the DNA sequence has four major roles: it can be </p>

<ol>
<li>a <em>regulatory gene</em> that encodes an inhibitor/repressor protein</li>
<li>a <em>promoter</em> segment where the RNA polymerase can associate with DNA to start transcription</li>
<li>an <em>operator</em> where a specific protein can bind</li>
<li>a <em>structural gene</em> which will be transcribed into RNA and then translated into a protein. </li>
</ol>

<p>When the operon is “ON”, an RNA polymerase complex associates with the promoter area and transcribes DNA into RNA.  The region of the RNA that delineates the start of each structural gene can then serve as a point where a ribosome can assemble and start the process of translating a new protein.</p>

<p>In the absence of lactose, the expression of the <em>lac</em> operon is maintained in the “OFF” state (Figure 2, panel a); the regulatory gene, <em>lacI</em>, codes for a repressor protein, which is constantly being made.   As its name suggests, in the absence of an “inducer” the repressor protein blocks transcription of the structural genes by binding to the operator region of the operon.  Binding of the repressor protein to this DNA element forms a physical barrier, which blocks the association of the RNA polymerase with the promoter region of DNA.  </p>

<p>The interaction of a protein with a ligand always results in a conformational change.   The <em>lac</em> operon is switched “ON” when the conformation of the repressor protein is changed due to its interaction with a chemical molecule known as an <strong>inducer</strong>.  The interaction of the repressor protein with an inducer prevents it from binding to the operator region of DNA.  Thus, the physical barrier blocking the association of the RNA polymerase is removed and transcription of the structural genes can proceed.</p>

<p>The three enzymes produced by the <em>lac</em> operon are </p>

<ul>
<li>β-galactosidase</li>
<li>galactoside permease</li>
<li>transacetylase.</li>
</ul>

<p>Since we cannot simply look at a bacterium to count the number of protein molecules present in the absence or presence of various inducers, the simplest way to determine if the lac operon is “ON” or “OFF” is to measure the enzymatic activity of β-galactosidase, which is represented by the equation below:</p>

<p>lactose + H<sub>2</sub>O \( {\underset{β-galactosidase}\longrightarrow} \) glucose + galactose</p>

<p>The reaction arrow points in one direction because the reaction is functionally irreversible; in other words, the reverse reaction happens at a rate so slow as to be negligible.</p>

<p>In theory there are many ways that one might be able to determine the activity of the β-galactosidase enzyme (e.g. test for its presence).  For instance, one could monitor either the disappearance of substrate (lactose) or the appearance of either of the reaction products (glucose or galactose).  Unfortunately, monitoring the levels of substrates and products requires laborious analytical procedures.  Instead, biochemists often use artificial substrates that are either colored or when acted upon by an enzyme, produce a colored product.  These types of molecules are easy to detect and therefore easy to <strong><em>MISSING WORD</em></strong> and quantify.</p>

<p>For our experimental system, the colorless substrate ONPG, (<em>ortho</em>-nitrophenyl-β-D-galactoside) is hydrolyzed by β -galactosidase to produce the colorless galactose (D-galactopyranose) and a bright yellow compound, <em>o</em>-nitrophenolate (ONP).  The amount of ONP generated in a specific amount of time can easily be assayed using a spectrophotometer and the use of a standard curve allows one to convert an absorbance reading into the number of moles of ONP that have been made.  </p>

<p>ONPG + H<sub>2</sub>O \( {\underset{β-galactosidase}\longrightarrow} \) D-galactopyranose + ONP</p>

<p>FIGURE 3 HERE</p>

<p><strong>Figure 3.</strong>  The action of β-galactosidase on the artificial substrate,(<em>o</em>-nitrophenyl-β-D-galactoside, produces the yellow product <em>o</em>-nitrophenol.</p>

<p>The spectrophotometer passes a beam of monochromatic light through a solution and a 
photodetector on the other side of the solution determines the percentage of light that passes through the solution.  The amount of light that is absorbed is presented in units of absorbance or optical density (OD).  The more light that is absorbed by molecules present in a solution, the higher the optical density.  In order to monitor the amount of a colored solute in a solution one must know what wavelength(s) of light the molecule absorbs.  Therefore, knowing that ONP, the product of the enzymatic reaction, strongly absorbs wavelengths of light that are 415 nm is a helpful piece of information. </p>

<p>In this series of labs, you will measure or “assay” the activity of β-galactosidase by monitoring the <strong>rate</strong> of ONP produced (e.g. amount of ONP produced per unit time).  The amount of ONP produced is proportional to the increase in absorbance at 415 nm, which can be determined by using a spectrophotometer or 96-well plate reader.  The details of how to use the plate reader are found in Section E of your lab supplement.  Readings using the 96-well plate reader will be taken at time 0 and then after 15 minutes.  The rate of β-galactosidase activity is represented by ∆absorbance at 415 nm/∆time [units: ∆OD/minute].  (NOTE: the Greek letter “∆” is shorthand for change).</p>

<p>Enzymatic activity is usually expressed in “International Units” (IU), which is a more formal and informative descriptor than “∆OD units/minute”.  One definition that is often used for a unit is the amount of enzyme needed to produce 1 µmole of product per minute.  In order to convert readings obtained in ∆OD units to µmoles, one must make use of a standard curve, which provides a mathematical representation of the relationship between the OD units and the µmoles of product.  </p>

<h2>Materials and Methods</h2>

<h3>A. Overview</h3>

<p>To test whether a compound can turn the expression of the <em>lac</em> operon “ON”, you will add it to an actively growing culture of bacteria.  After a specific amount of time, you will blow the bacteria apart to release all their innards (e.g. components of their cytosol), including any β-galactosidase that has been produced since the addition of the compound.  An increase in the amount of β-galactosidase would indicate the expression of the <em>lac</em>  operon.  In order to determine whether any β-galactosidase has been produced, you will add ONPG to the suspension, incubate this mixture for a set amount of time, and then measure the absorbance of the mixture at 415 nm.  You will then use a standard curve to determine the µmoles of ONP that were produced, and ultimately how much β-galactosidase is present. </p>

<h3>B.  Protocol</h3>

<p>This lab will meet for three weeks. </p>

<ul>
<li>Week 1: Create a standard curve, write an experimental design.</li>
<li>Week 2: Determine the enzymatic activity of β-galactosidase.</li>
<li>Week 3: Give oral report.</li>
<li>Week 4: Turn in written assignment.</li>
</ul>

<p><strong>FOR BUILDING A STANDARD CURVE</strong></p>

<p>You will need to prepare a standard curve to translate absorbance values into [ONP] in µM. You will then have to calculate the µmoles of ONP produced to calculate the amount of β-galactosidase present.</p>

<ol>
<li><p>Prepare a stock solution of ONP from powder. Calculate making 50 ml of a 2 mM ONP (MW=139.11) solution using 10 mM Tris Buffer, pH9.0. and have it checked by a TA before proceeding.</p></li>
<li><p>Set up a standard curve using your stock solution, with values ranging from 0-2000 µM, with the goal of having enough measurable data points (not too high or low to be read by the spectrophotometer) to draw a line. You are measuring different amounts of pure ONP dissolved in a solvent. Do you have a blank? Do you need one? What will you do with it?</p></li>
</ol>

<p>We suggest you make 0.25 ml of various dilutions of ONP using the 10 mM Tris buffer (pH 9.0) as diluent; a possible working table follows. (Will you write down your calculations? Where?)</p>

<table border = 1pt>
<tr><td> Well identifier in 96-well plate (e.g. A-2) <td>   1.5 ml µfuge tube label <td>   ONP Concentration<td>   µl 2 mM ONP stock <td> µl of 10 mM Tris (pH 9.0) <td> Final volume (VT) <td>  A<sub>415</sub>  </tr> </table>

<ol>
<li><p><strong>Each person</strong> should also obtain a solution of <strong>unknown</strong> amount of ONP.</p></li>
<li><p>Transfer 200 µl of each sample to a 96 well plate and read the absorbance at 415 nm.</p></li>
</ol>

<h3>EXPERIMENTAL DESIGN</h3>

<p>The detection of the synthesis of β-galactosidase requires bacteria that have a functional <em>lac</em> operon (Lac<sup>+</sup>), an inducer, and a substrate that when split by the enzyme produces a colored product, which can be measured by a spectrophotometer.</p>

<p>The interpretation of the data that you generate will be HEAVILY dependent on the inclusion of proper controls.  For instance, what if you treat a culture with a particular compound and find that there is no β-galactosidase activity present after the incubation.  What can you conclude?  How do you know if the bacteria possess a functional <em>lac</em> operon?  Conversely, if you detected high levels of β-galactosidase activity, how will you be sure that only the compound you are testing and nothing else is responsible for any observed increase in β-galactosidase expression?  Finally, how will you <em>statistically</em> analyze the results that you obtain?  What type of test(s) will you run and how many  trials are required to validate the use of the test you will use? <em>NOTE: Sections K and M of your lab supplement describes a number issues to consider when planning your experiments.</em></p>

<p><strong>SAFETY ISSUES:</strong>  </p>

<ol>
<li><p>Be careful when handling bacteria.  Although  billions of <em>E. coli</em> grow in the gut of your stomach, you should exercise care to avoid spilling or contaminating pipetmen or lab benches with <em>E. coli</em>. </p></li>
<li><p>Upon the completion of the lab, standard lab practice dictates that you wash your  hands with soap and water.</p></li>
<li><p>We will be using small amounts of organic solvent (chloroform) during the protocol.  Please use caution when pipeting or handling this solution.</p></li>
</ol>

<h3>PROTOCOL</h3>

<p><strong>FOR DETERMINING THE ENZYMATIC ACTIVITY OF β-GALACTOSIDASE</strong></p>

<ol>
<li><p>Pipet 0.9 mL of a log phase culture of <em>E. coli</em> into a set of clearly labeled glass test tube.</p>

<p><em>NOTE: These bacteria have been grown in tryptone broth, which has no sugars available.  The number of test tubes will depend on the number of inducers you will be testing and number of replicates your group would like to perform to facilitate statistical analysis of your results.</em></p></li>
<li><p>Add 0.1 mL of potential inducer, being sure to take note of the <em>final</em> concentration of the compound you are adding. </p></li>
<li><p>Mix and incubate the tubes in a 37&deg;C shaking water bath for 30 minutes.</p></li>
<li><p>Add 50 µL of CHCl<sub>3</sub> (chloroform), and 20 µL of 0.1% SDS (sodium dodecyl sulfate) to each sample and mix vigorously. </p>

<p><em>NOTE: The CHCl<sub>3</sub> kills the cells and disrupts the membranes, but does not affect the activity of the β-galactosidase enzyme. The SDS is a detergent that further disrupts the membranes.</em></p></li>
<li><p>Add 0.2 mL of 13 mM ONPG solution to each solution and RAPIDLY (but un-hurriedly) proceed to step 6 and 7.</p>

<p><em>NOTE:  Before adding the ONPG BE SURE that you and other members of your group are poised to complete steps # 6 and 7 in an efficient manner!</em></p>

<ul>
<li> Are there an appropriate number of tubes labeled?<br/></li>
<li> Does each of the labeled tubes contain 600 µL of 0.4 M Na<sub>2</sub>CO<sub>3</sub>?</li>
<li> Are the tubes arranged in a logical manner to minimize confusion? </li>
<li> Do you have an ice bucket ready?</li>
<li> Is there a micropipettor set to pipet 600 µL?</li>
<li> Do you have enough volume of the 0.4 M Na<sub>2</sub>CO<sub>3</sub>?</li>
<li> Is there a 37°C water bath running with enough room for your sample tubes?</li>
</ul></li>
<li><p>Using a P1000 set to pipet 600 µL, carefully ensure the samples are well mixed by smoothly pipetting up and down several times.</p></li>
<li><p>Transfer 0.6 mL of the reaction mixture to a separate, clearly labeled test tube that contains 600 µL of 0.4 M Na<sub>2</sub>CO<sub>3</sub>.  Place on ice.</p>

<p><em>NOTE: These tubes will serve as “Zero” time points.  The Na<sub>2</sub>CO<sub>3</sub> serves to stop the reaction and ensures the pH of the solution is basic, allowing the yellow color to fully develop.</em></p></li>
<li><p>Place the original sample tubes (which should contain 600 µL of solution) in a 37&deg;C shaking water bath for 15 minutes before proceeding to step 9.</p></li>
<li><p>Add 0.6 mL of 0.4 M Na<sub>2</sub>CO<sub>3</sub> to each sample tube and mix well.</p>

<p><em>NOTE: The Na<sub>2</sub>CO<sub>3</sub> serves to stop the reaction and ensures the pH of the solution is basic, allowing the yellow color to fully develop.</em></p></li>
<li><p>Allow the chloroform to settle to the bottom of the tube (5 minutes) and then transfer 200 µL from the very top of each sample tube a well of a 96-well plate.</p>

<p><em>NOTE: IT IS CRITICAL to avoid the cell residue and CHCl3 in the bottom of the tubes as they will strongly absorb light of 415 nm.  When transferring, take care to ensure that  your pipet tip is above this denser organic layer at the bottom.</em></p></li>
<li><p>Using the plate reader as your spectrophotometer, determine the A<sub>415</sub>  of each sample, the absorbance maximum for the orthonitrophenolate ion. (This reading is in optical density units, or OD units.)</p>

<p><em>NOTE: Be sure to make a table that represents the wells of the 96-well plate and record which sample you have placed in which position.  Notice that the rows of the plate are designated by a letter (A-H) and columns of the plate are designated with a number (1-12).  Thus, each position, just like the game Battleship, has coordinates such as “A-2”.  This is a handy way to keep track of the large number of samples you will be handling.</em></p></li>
<li><p>After recording your measurements, perform a series of calculations that allow you to analyze, interpret, and communicate your observations (e.g represent the amount of β-galactosidase activity in average (+/- S.D) IUs/sample, p-values from statistical tests, etc.)</p></li>
</ol>

<h3>Clean-up:</h3>

<ol>
<li><p>Empty all test tubes into the sink and throw them in glass disposal box.</p></li>
<li><p>Rinse beakers and all reusable glassware and put in brown tray near sink for dirty dishware.</p></li>
<li><p>SAVE 96-well plate for the next session, but don&#39;t rinse it!</p></li>
</ol>

<h3>Available possible inducers:</h3>

<p>Each compound comes as powder and should be dissolved in H<sub>2</sub>O as the solvent.  The molecular weights and structure of these molecules can be found in reference material such as </p>

<ul>
<li>the Merck Index  <a href="https://www.rsc.org/merck-index">https://www.rsc.org/merck-index</a> </li>
<li>the Aldrich chemical catalog  <strong><em>Is this accessible online without an account?</em></strong></li>
</ul>

<p>Choose one of the following possible inducers: </p>

<ul>
<li>10 mM glucose</li>
<li>10 mM sucrose (disaccharide of glucose and fructose)</li>
<li>10 mM galactose</li>
<li>10 mM alpha-lactose (alpha linkage form=not typical form found in nature)</li>
<li>10 mM fructose</li>
<li>10 mM maltose (disaccharide of 2 glucose)</li>
<li>10 mM starch (MW ???)</li>
<li>-starch is a string of 1-4 linkages of glucose, so treat as a glucose lacking an H2O molecule</li>
<li>10 mM cellobiose (disaccharide of 2 glucose, building block of cellulose)</li>
<li>10 mM deoxyribose</li>
<li>10 mM isopropyl-ß-D-thiogalactoside, or IPTG (a structural analog of lactose)</li>
<li>10 mM raffinose (trisaccharide of glucose, galactose and fructose) </li>
<li>10 mM saccharin</li>
</ul>

<h3>Other materials</h3>

<p>The following are already made for you:</p>

<ul>
<li>10 mM lactose (normal D-form with β-linkage)</li>
<li>0.4 M Na<sub>2</sub>CO<sub>3</sub></li>
<li>13 mM ONPG</li>
<li>CHCl<sub>3</sub> (chloroform)</li>
<li>0.1% SDS (sodium dodecyl sulfate)</li>
<li>  <em>Escherichia coli</em> (<em>E. coli</em>) - any wild type strain that is lac+</li>
<li>   Sterile water</li>
</ul>

<h3>Equipment</h3>

<ul>
<li> Shaking water bath at 37&deg;C</li>
<li> Spectrophotometer/Plate reader</li>
</ul>

<h2>Assignments</h2>

<h3>A. Standard Curve</h3>

<ol>
<li><p>Make a graph of your group’s data and perform a linear regression analysis to assess the relationship between ONP concentration and absorbance at 415 nm. This should be a complete figure with all the important features, including descriptive title and figure legend. Include the regression equation in the figure legend. </p></li>
<li><p>The Unknown sample you have been given is __________.  </p>

<ul>
<li>What is the A<sub>415</sub> value of your unknown sample? </li>
<li>How much ONP (µmoles or nmoles) is in the 200 µL of your unknown solution?<br/>

<ul>
<li>Combine your knowledge of spectrophotometry and standard curves to determine the amount. Show your thought process and calculations.</li>
</ul></li>
</ul></li>
<li><p>Each person should turn in this assignment individually, however, if you desire, you CAN do a single graph as a group, but you must acknowledge your group members by stating their names.</p></li>
</ol>

<h3>B. Experimental Design</h3>

<p>The week before you perform this experiment your group will write and turn in an experimental design.  The experimental design should be a rigorous and complete description of your plan as described in Section M of the lab supplement entitled “Scientific Investigation”.  </p>

<p>The design should clearly indicate: </p>

<ul>
<li><p>A hypothesis, and an explanation to justify your hypothesis  </p></li>
<li><p>A description of the samples (conditions) that will be included [+/- inducer(s)] accompanied by a brief explanation of why each is included</p>

<p>Since you will make up the solutions from powder of the compounds you test as inducers, please be sure to include the calculations for preparing 4 mL of the appropriate concentration of your potential inducers.</p></li>
<li><p>A  protocol with specific volumes, concentrations, times of incubation, stats test to use for analysis, number of replicates for each sample (we suggest 4 minimum), etc. </p></li>
</ul>

<h3>C.  Oral Presentation</h3>

<p>Following the collection of your data, your group will give an oral presentation on the results and interpretation of results of your experiment. Refer to the handout in your lab manual supplement on &ldquo;Oral Presentations&rdquo; for suggestions. This will most likely be a slide presentation using Powerpoint.</p>

<p>Some things to consider when writing the sections of your talk:</p>

<ul>
<li><p>Introduction</p>

<p>Include background so that audience knows the context and purpose of the experiment before you go into details in the next section.  Get to the point quickly!</p></li>
<li><p>Materials and Methods</p>

<p>Include the important details of how you performed the experimental procedures (e.g. the number of trials performed, what wavelength you read your samples at, the pH and molar concentration of buffers).</p>

<p>Do not include minute details of the experimental design that are irrelevant to the experiment’s outcome (e.g. which wells your solutions were plated in, how often you changed your pipet tip…). If you are bored speaking about it, it’s too detailed!</p></li>
<li><p>Results  </p>

<p>Include detailed figures (descriptive figure headings below the figures) and some calculations showing how you got your “IU” values. One of the  figures should be your standard curve, (even if you use mine for your calculations.) </p>

<p>Statistics are essential in supporting your conclusions.</p></li>
<li><p>Discussion</p>

<p>Be as broad-ranging as you can: Why do you think your potential inducer did/did not work? What experiments would you do in the future to confirm your conclusion or test additional questions? </p></li>
<li><p>References </p>

<p>A slide at the end to show the sources of your information.</p></li>
</ul>

<h3>D.  Written Report: An Abstract and Figure</h3>

<p>The abstract and graph are separate assignments. Information on one should NOT be assumed to be known on the other. </p>

<ol>
<li><p><em>Written abstract</em> 
Report on the results of this experiment. Refer to the handout in your lab manual supplement called &ldquo;Writing an Abstract&rdquo; for details. Please print your abstract on the &ldquo;blank&rdquo; abstract form in your lab manual supplement. Don’t scrimp on information regarding your data, data analysis (e.g. means +/- standard devaiation, appropriate statistical tests), and future experiments that you would propose! Please remember that all organisms identified by genus and species are italicized (example: <em>E. coli</em>).</p></li>
<li><p><em>Graph</em> </p>

<p>Please include a bar graph which summarizes your results on a separate page. Make it a complete figure with a title and an appropriate figure legend. Because the figure will represent “summary statistics” you should include error bars and, when appropriate, asterisks to indicate a level of statistical significance. </p></li>
</ol>

</body>

</html>

